components/sysutils/nut/nut-bins.p5m

```diff
@@ -0,0 +1,111 @@
+
+file path=usr/bin/$(MACH64)/nut-scanner
+file path=usr/bin/$(MACH64)/upsc
+file path=usr/bin/$(MACH64)/upscmd
+file path=usr/bin/$(MACH64)/upslog
+file path=usr/bin/$(MACH64)/upsrw
+
+file path=usr/bin/nut-scanner
+file path=usr/bin/upsc
+file path=usr/bin/upscmd
+file path=usr/bin/upslog
+file path=usr/bin/upsrw
+
+file path=usr/sbin/$(MACH64)/upsd
+file path=usr/sbin/$(MACH64)/upsdrvctl
+file path=usr/sbin/$(MACH64)/upsmon
+file path=usr/sbin/$(MACH64)/upssched
+
+file path=usr/sbin/upsd
+file path=usr/sbin/upsdrvctl
+file path=usr/sbin/upsmon
+file path=usr/sbin/upssched
+
+# Example notification script for UPSMON
+file files/notifyme path=usr/lib/nut/bin/notifyme mode=0755 preserve=renamenew
+
+# This is an example script for handling upssched
+file path=usr/bin/upssched-cmd preserve=renamenew
+
+# Init-scripts for NUT, not perfect - but better than nothing for a start.
+# NUT should start generally after networking (if SNMP or NetXML is used,
+# or if this box is a client to remote upsd). Ending is trickier - could
+# want it to issue killpower late in shutdown, but have network to be able
+# to do so (for remote UPSes). This can be tuned in XML manifest by user.
+file files/svc-nut path=lib/svc/method/svc-nut
+file files/nut.xml path=var/svc/manifest/application/nut.xml
+
+file files/svc-nut-client path=lib/svc/method/svc-nut-client
+file files/nut-client.xml path=var/svc/manifest/application/nut-client.xml
+
+file path=usr/share/man/man5/hosts.conf.5
+file path=usr/share/man/man5/ups.conf.5
+file path=usr/share/man/man5/upsd.conf.5
+file path=usr/share/man/man5/upsd.users.5
+file path=usr/share/man/man5/upsmon.conf.5
+file path=usr/share/man/man5/upssched.conf.5
+
+file path=usr/share/man/man8/nut-scanner.8
+file path=usr/share/man/man8/upsc.8
+file path=usr/share/man/man8/upscmd.8
+file path=usr/share/man/man8/upsd.8
+file path=usr/share/man/man8/upsdrvctl.8
+file path=usr/share/man/man8/upslog.8
+file path=usr/share/man/man8/upsmon.8
+file path=usr/share/man/man8/upsrw.8
+file path=usr/share/man/man8/upssched.8
+
+# We deliver the sample configs as the ones to use,
+# at worst they have no actual information inside and do nothing.
+file etc/nut/ups.conf.sample path=etc/nut/ups.conf \
+    group=nut owner=root mode=0640 preserve=true
+file etc/nut/upsd.conf.sample path=etc/nut/upsd.conf \
+    group=nut owner=root mode=0640 preserve=true
+file etc/nut/upsd.users.sample path=etc/nut/upsd.users \
+    group=nut owner=root mode=0640 preserve=true
+file etc/nut/upsmon.conf.sample path=etc/nut/upsmon.conf \
+    group=nut owner=root mode=0640 preserve=true
+file etc/nut/upssched.conf.sample path=etc/nut/upssched.conf \
+    group=nut owner=root mode=0640 preserve=true
+
+file build/$(MACH32)/scripts/avahi/nut.service path=$(NUT_AVAHI_SERVICE_SAMPLE)
+
+# TODO pedantic: split this off to depend on logrotate package?
+file scripts/logrotate/nutlogd path=etc/logrotate.d/nutlogd preserve=renamenew
```

Comment:
We don't need it or logadm entry, as nut daemons logs messages to daemon facility,. and these logs go to /var/adm/messages.